mcux/mcux-sdk/drivers/rtwdog/fsl_rtwdog.c

```diff
@@ -144,5 +144,8 @@ void RTWDOG_Deinit(RTWDOG_Type *base)
     primaskValue = DisableGlobalIRQ();
     RTWDOG_Unlock(base);
     RTWDOG_Disable(base);
+    while ((base->CS & RTWDOG_CS_RCS_MASK) == 0U)
```

Comment:
@SuperHeroAbner has this change been integrated into the SDK?

Reply:
Hi @dleach02 , not in develop/release SDK now. I just found this when support Zephyr. I plan to integrate it into 25.3 SDK release.